tests/test_apps.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from django.apps import apps
+from django.db import models
+from django.core.exceptions import ImproperlyConfigured
+from django.test import override_settings
+from vitrina.utils import get_all_models, is_model_versioned
+from django.contrib import admin
+from vitrina.admin import RevisionCommentVersionAdmin
+
+import reversion
+
+
+@pytest.mark.parametrize(
+    "model, should_be_registered",
+    [
+        (model, is_model_versioned(model))
+        for model in get_all_models()
+    ],
+)
+def test_reversion_model_registration(model: type[models.Model], should_be_registered: bool):
+    path = f"{model.__module__}.{model.__name__}"
+
+    if should_be_registered:
+        assert reversion.is_registered(model), (
+            f"Model '{path}' should be registered with reversion."
+        )
```

Comment:
Line length is 120, should be more than enough to put in one line
```suggestion
        assert reversion.is_registered(model), f"Model '{path}' should be registered with reversion."
```